executor/src/witgen/affine_expression.rs

```diff
@@ -374,11 +367,21 @@ mod test {
 
     use super::*;
     use crate::witgen::{bit_constraints::BitConstraint, EvalError};
-    use number::FieldElement;
+    use number::GoldilocksField;
+
+    impl<K> std::ops::Mul<AffineExpression<K, GoldilocksField>> for GoldilocksField {
```

Comment:
Doesn't it have that already?

Reply:
No this one is tricky: https://github.com/rust-lang/rust/blob/master/compiler/rustc_error_codes/src/error_codes/E0210.md
So it didnt work generically so since it was only used in tests I fixed it for Goldilocks

Reply:
We have the reverse implementation though, so `k * e` could be replaced by `e * k` I think